content/applications/finance/accounting/customer_invoices.rst

```diff
@@ -4,225 +4,230 @@
-| **Account**           | **Partner**   | **Due date**   | **Debit**   | **Credit**   |
-+=======================+===============+================+=============+==============+
-| Bank                  | Agrolait      |                | 115         |              |
-+-----------------------+---------------+----------------+-------------+--------------+
-| Accounts Receivable   | Agrolait      |                |             | 115          |
-+-----------------------+---------------+----------------+-------------+--------------+
-
-Receive a partial payment through the bank statement
-----------------------------------------------------
+Payment follow-up
+=================
 
-You can manually enter your bank statements in Odoo, or you can import
-them in from a csv file or from several other predefined formats
-according to your accounting localisation.
+Odoo helps define a :doc:`follow-up <payments/follow_up>` strategy. Different actions can be set up
+to remind customers to pay their outstanding invoices, depending on how much the customer is
+overdue. These actions are bundled into follow-up levels that trigger when an invoice is overdue by
+a certain number of days. If there are other overdue invoices for the same customer, the actions are
+performed on the most overdue invoice.
 
-Create a bank statement from the accounting dashboard with the related
-journal and enter an amount of $100 .
+.. _accounting/invoice/followup-reports:
 
-.. image:: customer_invoices/invoice06.png
+Follow-up Reports
+-----------------
 
-Reconcile
----------
+To set the different follow-up levels, go to
+:menuselection:`Accounting --> Configuration --> Follow-up Levels`. All actions triggered after a
+specified number of days can be viewed. To manage the various options, such as the number of days,
+the type of reminder sent, or whether the action is automated, click on the specific line that needs
+to be updated.
 
-Now let's reconcile!
+Then, follow these steps to take the appropriate action:
 
-.. image:: customer_invoices/invoice07.png
+- To access the list of customers requiring action along with the :guilabel:`Total Due` and
+  :guilabel:`Total Overdue` amounts, go to
+  :menuselection:`Accounting --> Customers --> Follow-up Reports`.
+- To get a full overview of the customer's open invoices, click on its line.
+- To open a payment reminder to be sent to the customer, click the :guilabel:`Follow up` button.
+- To exclude any invoices from the follow-up, enable the option :guilabel:`Exclude from Follow-ups`
+  for the selected ones.
 
-You can now go through every transaction and reconcile them or you can mass reconcile with instructions at the bottom.
+.. _accounting/invoices/aging-report:
 
-After reconciling the items in the sheet, the related invoice will now
-display "You have outstanding payments for this customer. You can
-reconcile them to pay this invoice. "
+Aged Receivable
```

Comment:
I understand why this could belong under Payment Follow-ups, but I think it would fit better in the Reporting section and could be paired with the Aged Payable and Partner Ledger. Let's have a "Partner Reports" section under "Reporting" that explains the three reports briefly. It could be as short as one line per report like the P&L and BS have or a little longer if you have more to say (since they don't have their own pages like the P&L and BS).